pilot/pkg/model/validation.go

```diff
@@ -1319,8 +1319,10 @@ func validateTLSMatch(match *networking.TLSMatchAttributes) (errs error) {
 	if len(match.SniHosts) == 0 {
 		errs = appendErrors(errs, fmt.Errorf("TLS match must have at least one SNI host"))
 	}
-	if match.DestinationSubnet != "" {
-		errs = appendErrors(errs, ValidateIPv4Subnet(match.DestinationSubnet))
+	if len(match.DestinationSubnet) > 0 {
```

Comment:
in the api, shouldn't this ideally be `match.DestinationSubnets`?

Reply:
Ah dang!!